server/src/main/webapp/verifier.html

```diff
@@ -19,8 +19,8 @@
     </header>
 
     <main>
-        <h1 class="text-body-emphasis">Request Verified Identity Documents</h1>
-        <p class="fs-5 col-md-8">Request a verified identity document such as a Mobile Driving License (mDL) or EU PID</p>
+        <h1 class="text-body-emphasis">Request Verified Documents</h1>
```

Comment:
hmm if we're looking at the wording really critically, what does "verified" document really mean? I think it's a little hard to understand for an average user

Reply:
Rephrased to Request Digital Documents, for now.